cli/internal/packagemanager/yarn.go

```diff
@@ -42,6 +48,13 @@ var nodejsYarn = PackageManager{
 
 		globs, err := pm.getWorkspaceGlobs(rootpath)
 		if err != nil {
+			// In case of a non-monorepo, the workspaces field is empty and only node_modules in the root should be ignored
+			_, ok := err.(*NoWorkspacesFoundError)
+			if ok {
+				ignores := make([]string, 1)
+				ignores[0] = "node_modules/**"
+				return ignores, nil
```

Comment:
```suggestion
				return []string{"node_modules/**"}, nil
```

(I'm not a Go expert here, but this reads a bit cleaner to me. I'm not sure if the size allocation is implicit when declaring with literal values, or if your `make` with size is better. cc @chris-olszewski